showpass/Showpass-E2E-tests
# TA-50 Verify package order receipt page

**Description:**
This spec will be verifying a particular ticket package purchase. This order is linked to the following user:  [user-packages@mailinator.com](mailto:user-packages@mailinator.com)

- Navigate to https://beta.showpass.com/account/my-orders/
- Find the following order by Order ID: 1e-d67b-428a-9748-d638af16e987
- Make sure the order details are correct

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-50

**How to test:**
Run the following test suite and ensure the tests pass:
cypress/e2e/user-settings/my-orders.cy.js

**Test execution results:**

<img width="1712" alt="AT-50-test-passed" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/48529c6b-1e62-4973-9130-b517a59786d2">
